integrations/access/servicenow/client.go

```diff
@@ -129,6 +129,7 @@ func NewClient(conf ClientConfig) (*Client, error) {
 		SetHeader("Content-Type", "application/json").
 		SetHeader("Accept", "application/json").
 		SetBasicAuth(conf.Username, conf.APIToken)
+	client.OnAfterResponse(common.OnAfterResponse("servicenow", errWrapper, conf.StatusSink))
```

Comment:
Nit: Prefer to use existing constants:
```suggestion
	client.OnAfterResponse(common.OnAfterResponse(types.PluginTypeServiceNow, errWrapper, conf.StatusSink))
```